### PR TITLE
Increased box size.

### PR DESCRIPTION
**Pull request by SreelaxmiAjayan**
Increased box size along with being responsive.

### CODE REVIEW

File: src/app/components/SubSection.tsx [hunk 1]
```diff
@@ -80,14 +81,15 @@ export default function SubSection() {
                 display={"flex"}
                 justifyContent={"space-evenly"}
                 alignItems={"center"}
-                width={"100%"}
-                px={48}
-                py={6}
+               
                 sx={{
-                    backgroundColor: "lightgray",
+                    backgroundColor: "#f6f6f6",
```
Comment (sharan-experion) [inline src/app/components/SubSection.tsx:86]: Do not add a hard coded value here, change the value of light gray inside the custom theme file

File: src/app/components/OtherSections.tsx [hunk 2]
```diff
@@ -56,39 +53,47 @@ export default function OtherSections({
...
-    <ImageButton
-      focusRipple
-      key={label}
-      style={{
-        width: "30%",
-        height: "30%",
-        aspectRatio: 1,
-        backgroundColor: "black",
-        textAlign: "left",
+    <Box
```
Comment (sharan-experion) [inline src/app/components/OtherSections.tsx:56]: None of these style changes fix the issue of responsiveness. Please test for yourself.

File: src/app/components/SubSection.tsx [hunk 3]
```diff
@@ -39,16 +39,20 @@ export default function SubSection() {
                 gap={6}
                 width={"100%"}
                 my={4}
```
Comment (sharan-experion) [inline src/app/components/SubSection.tsx:41]: This margin is being overridden by some other style rule.  This css selector will vary for each build, but inspecting this div will show this as the first style rule. Please remove that overriding style rule so this margin will work.
image

File: src/app/components/SubSection.tsx [hunk 4]
```diff
@@ -70,24 +74,27 @@ export default function SubSection() {
...
+                        
                     }}
                 >
                     VIEW ALL RESULTS
                 </Button>
-
             </Box>
             <Box
                 display={"flex"}
                 justifyContent={"space-evenly"}
```
Comment (sharan-experion) [inline src/app/components/SubSection.tsx:85]: Change this to center the content as per the source design.

File: src/app/components/SubSection.tsx (hunk 3 above)
Comment (SreelaxmiAjayan) [reply src/app/components/SubSection.tsx]: Style fix done.

File: src/app/components/SubSection.tsx (hunk 4 above)
Comment (sharan-experion) [reply src/app/components/SubSection.tsx]: This is not a fix, this a workaround by setting hard coded values. Please address it properly.

File: src/app/components/SubSection.tsx (hunk 3 above)
Comment (sharan-experion) [reply src/app/components/SubSection.tsx]: This is not a fix, this a workaround by setting hard coded values. Please address it properly.

File: src/app/components/SubSection.tsx (hunk 4 above)
Comment (sharan-experion) [reply src/app/components/SubSection.tsx]: Please remove the margin on the buttons and use the gap property on the flex container to control the spacing between the boxes.

**Review by sharan-experion — APPROVED** [review]

**Pull request merged**